app/components/subscription/prices.tsx

```diff
@@ -47,56 +55,97 @@ export const Price = ({
   const activePlan = activeSubscription?.items.data[0]?.plan;
   const isFreePlan = price.id != "free";
   const features = price.product.metadata.features?.split(",") || [];
+
+  // icons to be mapped with different plans
+  const plansIconsMap: { [key: string]: JSX.Element } = {
+    free: <SingleLayerIcon />,
+    tier_1: <DoubleLayerIcon />,
+    tier_2: <MultiLayerIcon />,
+  };
+
+  function shortenIntervalString(str: string): string {
```

Comment:
This is nice, but can you move it to a util file and import it from there. i dont like defining it inside the component.